packages/lit-element/src/lib/updating-element.ts

```diff
@@ -422,13 +422,17 @@ export abstract class UpdatingElement extends HTMLElement {
     );
   }
 
+  protected static get hasFinalized() {
+    return this.hasOwnProperty(finalized);
```

Comment:
How about:

```ts
this[finalized] = false;
```

The we shouldn't need hasOwnProperty checks?

Reply:
Finalize has to be done on each class in the prototype chain. This is why there's an own property check.